include/cantera/zeroD/ReactorNet.h

```diff
@@ -222,6 +237,20 @@ class ReactorNet : public FuncEval
         return m_integ->maxSteps();
     }
 
+    //! Returns the order used during the last solution step
+    virtual int lastOrder() {
+        return m_integ->lastOrder();
+    }
+
+    //! Set absolute tolerances during advanceTowards
+    virtual void setAtolAdvance(const double* atols);
```

Comment:
I'm a little ambivalent about the re-use of the term "tolerances" for this new feature, in addition to the absolute integrator tolerances. Would it make sense to refer to these as "thresholds", or something else?

Reply:
I have renamed this function multiple times thus far and will settle on any good suggestion.

Reply:
Will be renamed to `advance_limit`/`set_advance_limit`, per discussion.